python/graphframes/graphframe.py

```diff
@@ -102,6 +111,102 @@ def bfs(self, fromExpr, toExpr, edgeFilter=None, maxPathLength=10):
         jdf = builder.run()
         return DataFrame(jdf, self._sqlContext)
 
+    # Standard algorithms
+
+    def connected_components(self):
+        """
+        Computes the connected components of the graph.
+        :return:
+        """
+        jgf = self._jvm_graph.connectedComponents().run()
+        return _from_java_gf(jgf, self._sqlContext)
+
+    def label_propagation(self, max_steps):
```

Comment:
For these methods, shall we use the keyword_only decorator from from pyspark.ml.util (copying it in, not using the pyspark decorator)?